microbosh-setup/micro_cf.yml

```diff
@@ -0,0 +1,375 @@
+    staging_upload_password: c1oudc0w
+    staging_upload_user: staging_upload_user
+  ccdb:
+    address: 10.0.16.4
+    databases:
+    - {name: ccdb, tag: cc}
+    db_scheme: postgres
+    port: 5524
+    roles:
+    - {name: ccadmin, password: c1oudc0w, tag: admin}
+  databases:
+    address: 10.0.16.4
+    databases:
+    - {name: ccdb, tag: cc, citext: true}
+    - {name: uaadb, tag: uaa, citext: true}
+    port: 5524
+    roles:
+    - {name: ccadmin, password: c1oudc0w, tag: admin}
+    - {name: uaaadmin, password: c1oudc0w, tag: admin}
+  dea_next:
+    advertise_interval_in_seconds: 5
+    heartbeat_interval_in_seconds: 10
+    memory_mb: 33996
+  description: Cloud Foundry sponsored by Pivotal
+  domain: cf.azurelovecf.com
+  etcd:
+    machines: [10.0.16.4]
+  hm9000:
+    url: http://hm9000.cf.azurelovecf.com
+  logger_endpoint:
+    port: 4443
+  loggregator:
+    incoming_port: 13456
+    dropsonde_incoming_port: 13457
+    outgoing_dropsonde_port: 18081
+    doppler_port: 8081
+  loggregator_endpoint:
+    shared_secret: c1oudc0w
+  login:
+    protocol: http
+  metron_agent:
+    zone: z1
+    status:
+      user: trafic_controller
+      password: trusty
+      port: 6799
+    incoming_port: 3456
+    dropsonde_incoming_port: 3457
+  metron_endpoint:
+    shared_secret: trusty
+    host: 127.0.0.1
+    port: 3456
+    dropsonde_port: 3457
+  nats:
+    machines: [10.0.16.4]
+    password: c1oudc0w
+    port: 4222
+    user: nats
+    debug: true
+    trace: true
+  nfs_server:
+    address: 10.0.16.4
+    allow_from_entries: [10.0.16.0/20]
+  ssl:
+    skip_cert_verify: true
+  system_domain: cf.azurelovecf.com
+  system_domain_organization: default_organization
+  uaa:
+    clients:
+      cc-service-dashboards:
+        authorities: clients.read,clients.write,clients.admin
+        authorized-grant-types: client_credentials
+        scope: openid,cloud_controller_service_permissions.read
+        secret: c1oudc0w
+      cloud_controller_username_lookup:
+        secret: c1oudc0w
+      doppler:
+        authorities: uaa.resource
+        secret: c1oudc0w
+      login:
+        authorities: oauth.login,scim.write,clients.read,notifications.write,critical_notifications.write,emails.write,scim.userids,password.write
+        authorized-grant-types: authorization_code,client_credentials,refresh_token
+        redirect-uri: http://login.cf.azurelovecf.com
+        scope: openid,oauth.approvals
+        secret: c1oudc0w
+      servicesmgmt:
+        authorities: uaa.resource,oauth.service,clients.read,clients.write,clients.secret
+        authorized-grant-types: authorization_code,client_credentials,password,implicit
+        autoapprove: true
+        redirect-uri: http://servicesmgmt.cf.azurelovecf.com/auth/cloudfoundry/callback
+        scope: openid,cloud_controller.read,cloud_controller.write
+        secret: c1oudc0w
+    jwt:
+      signing_key: |
+        -----BEGIN RSA PRIVATE KEY-----
```

Comment:
I have double checked with abel, he said this ssl is common to bosh development. I have his promise , so it's ok to check in them @AbelHu